code/game/machinery/constructable_frame.dm

```diff
@@ -1670,3 +1670,18 @@ to destroy them and players will be able to make replacements.
 		/obj/item/weapon/stock_parts/manipulator = 3,
 		/obj/item/weapon/stock_parts/micro_laser = 1
 	)
+
+/obj/item/weapon/circuitboard/suit_storage_unit
+	name = "Circuit board (Suit Storage Unit)"
+	desc = "A circuit board used to clean, charge, and store a hardsuit"
+	build_path = /obj/machinery/suit_storage_unit
+	board_type = MACHINE
+	origin_tech = Tc_ENGINEERING + "=2;" + Tc_PROGRAMMING + "=2" + Tc_POWERSTORAGE + "=2"
```

Comment:
missing ;

Reply:
Where does the semicolon go here?
The two boards above it have only 1 semicolon in the tech string
Seemed a bit weird to me, but I just went with what was there.

Should each tech be separated by one? (Future PR to fix the others?

Reply:
origin_tech is supposed to be a list that looks like this:

"engineering=2;programming=2;powerstorage=2"
; separates each entry

there's no ; after programming in this one

Reply:
added